syntax = "proto3";

package raw_data;

message InstrumentCurrencyVolRequest {
    string symbol = 1;
}

message InstrumentCurrencyVolResponse {
    bytes series = 1;
}

service InstrumentCurrency {
    rpc get_instrument_currency_vol (InstrumentCurrencyVolRequest) returns (InstrumentCurrencyVolResponse);
}